src/components/network/equipment-table-editors.js

```diff
@@ -86,8 +89,69 @@ export const NumericalField = ({
                     },
                     min: { min },
                     max: { max },
+                    step: 'any',
                 }}
             />
-        </Tooltip>
+        );
+    }
+
+    return (
+        <div>
+            {min !== undefined || max !== undefined ? (
+                <Tooltip
+                    title={intl.formatMessage({ id: 'MinMax' }, { min, max })}
+                >
+                    {renderNumericText()}
+                </Tooltip>
+            ) : (
+                renderNumericText()
+            )}
+        </div>
+    );
+};
+
+export const NameField = ({ defaultValue, setter, style, ...props }) => {
+    return (
+        <TextField
+            defaultValue={defaultValue}
+            onChange={(ev) => setter(ev.target.value)}
+            {...props}
+            size={'small'}
+            margin={'none'}
+            style={{ ...style, padding: 0 }}
+            inputProps={{
+                style: {
+                    textAlign: 'center',
+                    fontSize: 'small',
+                    flexGrow: 1,
+                },
+            }}
+        />
+    );
+};
+
+export const EnumField = ({ enumList, setter, defaultValue, ...props }) => {
+    return (
+        <Select
+            defaultValue={defaultValue || ''}
+            onChange={(ev) => setter(ev.target.value)}
+            size={'medium'}
+            margin={'none'}
+            MenuProps={{
```

Comment:
why is it needed ?

Reply:
I dont know. I copy/paste this part from another Select Editor to look like the same. To be verified

Reply:
I got the answer: it was to limit the height of the select one opened. I dont need it in my "enum" case, so I'll remove it.